### PR TITLE
Leveraging OpenAI Gym and the Anytrading Environment for Trading

### LINKED ISSUE #5973: [Machine Learning] Leveraging OpenAI Gym and the Anytrading Environment for Trading
#### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. 

Use keyword research to improve your article's SEO.

Leveraging OpenAI Gym and the Anytrading Environment for Trading 

#### Proposed article introduction
*We're looking for the first 2-3 paragraphs of your article that appropriately summarize what your article will be about.*
We will use this writing snippet to help us assess the overall quality before approval so ensure it has been proofread, spellchecked and grammarchecked.

AnyTrading is a collection of OpenAI Gym environments for reinforcement learning-based trading algorithms. AnyTrading aims to provide some Gym environments to improve and facilitate the procedure of developing and testing RL-based algorithms in this area. 
In this tutorial, we'll leverage Anytrading environment to build a RL agent that can trade.

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.

By the end of this tutorial, a reader will be able to understand:
- How to get started with Gym Anytrading environment
- How to install the environment
- How to load historical data from [MarketWatch](https://www.marketwatch.com/)
- Train an RL agent to trade using the Gym environment

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

#### References
Please list links to any published content/research that you intend to use to support/guide this article.
- [Gym Anytrading](https://github.com/AminHP/gym-anytrading)

### PR DESCRIPTION
**Pull request by Bayler**
This closes #5973

## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by WanjaMIKE** [comment]
Plagiarism check is done.

File: content/articles/leveraging-openai-gym-and-the-anytrading-environment-for-trading/index.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  - url: /engineering-education/leveraging-openai-gym-and-the-anytrading-environment-for-trading/hero.png
+    alt: OpenAI Gym and the Anytrading Environment Hero Image
+---
+This tutorial will show you how to leverage Reinforcement Learning for trading. 
+<!--more-->
+
+### Prerequisites
+To follow along with this tutorial, you need to be familiar with:
+- [Reinforcement Learning](engineering-education/introduction-to-reinforcement-learning/) and its algorithms.
+- Any kind of trading.
```
Comment (collins-a) [inline content/articles/leveraging-openai-gym-and-the-anytrading-environment-for-trading/index.md:23]: Specify the type/scope of trading you are referring to here as trading can vary depending on context

File: content/articles/leveraging-openai-gym-and-the-anytrading-environment-for-trading/index.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+```python
+model_train = A2C('MlpLstmPolicy', env, verbose=1) 
+model_train.learn(total_timesteps=100000)
+```
+With these two lines of code, our model will start training.
+
+`A2C` is the algorithm that we will use for this run. `A2C` is an acronym that stands for `Advantage Actor-Critic. It is an RL algorithm that combines Policy-Based and Value-Based RL techniques. You can read more about it [here](https://towardsdatascience.com/advantage-actor-critic-tutorial-mina2c-7a3249962fc8).
+We are using the `MlpLstmPolicy` which is a deep neural network policy with an LSTM layer. It is an important policy as it allows a neural network to keep context and learn about the previous history within its neurons.
+The last line of code kicks off training with `100000` timesteps. Ideally, what you need to observe while training is the `explained_variance` value. You want it to be as high as possible. We are looking at values between `0` and `1`. A number close to `1` denotes a high value while that close to `0`, a low value. You should also make sure the `value_loss` is as low as possible.
```
Comment (collins-a) [inline content/articles/leveraging-openai-gym-and-the-anytrading-environment-for-trading/index.md:188]: Revise the use of backticks in these two paragraphs to highlight the correct words

**Review by collins-a — COMMENTED** [review]
@Bayler great tutorial. A couple of minor considerations and we'll be ready for the final review

**Review by collins-a — APPROVED** [review]

**Comment by ahmadmardeni1** [comment]
Great work. @Bayler @collins-a

**Pull request merged**